### PR TITLE
Phase2-gex48 Add 2 new scenarios D75 and D76 using I13 for D71 and D74. Also remove some overlaps

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add 2 new scenarios D75 and D76 using I13 for D71 and D74. Also remove some overlaps

#### PR validation:

Tested using overlap testing tools and runTheMatrix test workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
@bsunanda was the overlap removal here expected to impact the older Phase 2 scenarios?

**Comment by bsunanda** [comment]
@kpedro88 The overlaps were quite serious. Yes it will. Should I stop removing overlaps and make a new path only for the new scenarios?

**Comment by kpedro88** [comment]
@bsunanda ultimately, what matters is the impact of these overlaps on physical results. If we still expect comparisons of new algorithms in 11_3_X to be valid for backporting those algorithms to 11_1_X (where we do not plan further updates to the D49 geometry), it is probably best to continue to fix D49 and other scenarios in the master branch.

**Comment by bsunanda** [comment]
@kpedro88 OK - in that case I leave them as they are now with overlap fixed for the 2026 scenarios in the master branch

**Comment by srimanob** [comment]
A question on overlap not sure I understand. If there is a large overlap between MUON and CALO, why did we miss them since D71 and D74 were added?
- D71 = T21+C14+M7+I11+O7+F6
- D74 = T21+C14+M9+I11+O7+F6

Do I miss a point here? Thanks very much @bsunanda @kpedro88

**Comment by bsunanda** [comment]
Overlaps are cured by changing muonBase.xml used in O4, O5, O6, O7. So it will affect all 2026 scenarios including D71 and D74. The 2 new scenarios: D75, D76 use I13 instead of I11.

File: Geometry/MTDCommonData/data/etl/v4/etl.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0"?>
-<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../../DetectorDescription/Schema/DDLSchema.xsd">
+<DDDefinition>
 
   <ConstantsSection label="etl.xml" eval="true/false">
-    <Constant name="ETLrmin" value="[caloBase:Rmin100]"/>   <!-- 26.3 cm -->
+    <Constant name="ETLrmin" value="[caloBase:Rmin100]"/>   <!-- 28.5 cm -->
     <Constant name="ETLrmax" value="[caloBase:Rmax100]"/>   <!-- 125.525 cm -->
-    <Constant name="ETLthickness" value="7.7"/>
```
Comment (fabiocos) [inline Geometry/MTDCommonData/data/etl/v4/etl.xml:7]: @bsunanda thanks for updating that, there was a cleaning of v5 (I13) but not of this earlier version

**Comment by kpedro88** [comment]
@bsunanda can you update the PR title?

**Comment by cvuosalo** [comment]
+1

**Comment by srimanob** [comment]
+Upgrade

**Comment by chayanit** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**